yt/visualization/base_plot_types.py

```diff
@@ -151,11 +150,20 @@ def _create_axes(self, axrect: tuple[float, float, float, float]) -> None:
 
     def _get_canvas_classes(self):
         if self.interactivity:
-            key = str(matplotlib.get_backend()).lower()
+            key = str(matplotlib.get_backend())
         else:
             key = "agg"
 
-        module, fig_canvas, fig_manager = BACKEND_SPECS[key]
+        if matplotlib.__version_info__ >= (3, 9):
+            # once yt has a minimum matplotlib version of 3.9, this branch
```

Comment:
I would not bother with this comment; to me, it's 100% implied by the code itself. But, NBD if you think it's helpful.

Reply:
mainly added it cause BACKEND_SPECS is not defined within the scope of the function. so i'd rather keep it i think? but also don't feel strongly about it.